lib/browser/api/menu.js

```diff
@@ -110,6 +110,10 @@ Menu.prototype.insert = function (pos, item) {
     throw new TypeError('Invalid item')
   }
 
+  if (pos > this.getItemCount()) {
```

Comment:
Purely because some devs might be doing strange math and getting bad indexes.  We should probably also check for `pos < 0` 🤔

Reply:
lol
good point tho, on it